src/mon/MonitorDBStore.h

```diff
@@ -662,6 +662,7 @@ class MonitorDBStore
     // there should be no work queued!
     io_work.stop();
     is_open = false;
+    db.reset(NULL);
```

Comment:
i think we'd better not to `db.reset()`. because, the counter part call of `close()` is `open()`, which just init the db and reset `is_open`.

but the only caller of `MonitorDBStore::close()` i found is ceph_mon.cc and ceph_monstore_tool.cc, neither of them is trying to reuse the db after closing it, though. so it's safe to do so.